rllab/envs/normalized_env.py

```diff
@@ -1,3 +1,4 @@
+import gym
```

Comment:
what is the relationship between this file and #125 ?

Reply:
This file will be replaced by #125, but the utility functions located in #125 are relocated here in rllab.envs.util. I will remove edits to this file.